src/import/FSHImporter.ts

```diff
@@ -795,8 +795,14 @@ export class FSHImporter extends FSHVisitor {
     return rules;
   }
 
-  private parseCard(card: string): { min: number; max: string } {
+  private parseCard(card: string, rule: CardRule): { min: number; max: string } {
     const parts = card.split('..', 2);
+    if (parts[0] === '' && parts[1] === '') {
+      logger.error(
+        `Neither side of the cardinality was specified on path "${rule.path}". A min, max, or both need to be specified.`,
+        rule.sourceInfo
+      );
+    }
     return {
       min: parseInt(parts[0]),
```

Comment:
What if here you did 
```
min: parseInt(parts[0]) || undefined,
```
Then the `min` would be undefined if no min was specified. This is only a suggestion and maybe there is some reason I don't see why this doesn't work. Even if this does work, I'm still willing to be convinced that leaving it as NaN is cleaner anyway, because it is not a number.

Reply:
Hm that's a good idea! I actually think that if I do this for the `min` and `max` here, then both values will consistently be `undefined` if they have no value specified. I think I like the consistency for the two values, so I'm going to update this and the tests.

Reply:
After attempting this, it appears that `max` doesn't easily work having an `undefined` value. I'll keep exploring these options.